panel/browser-panel-client.cpp

```diff
@@ -186,6 +194,73 @@ bool QCefBrowserClient::OnBeforePopup(
 	return true;
 }
 
+void QCefBrowserClient::OnBeforeContextMenu(CefRefPtr<CefBrowser>,
+					    CefRefPtr<CefFrame>,
+					    CefRefPtr<CefContextMenuParams>,
+					    CefRefPtr<CefMenuModel> model)
+{
+	if (model->IsVisible(MENU_ID_BACK) &&
+	    model->IsVisible(MENU_ID_FORWARD) &&
```

Comment:
You can probably only check one of these conditions, or check if "Copy" does not exist, since the intent was to not add the Reload button when text is highlighted.

Reply:
Done